tests/Feature/GithubWebhookControllerTest.php

```diff
@@ -0,0 +1,280 @@
+    protected $webhookController;
+    protected $organization;
+    protected $user;
+    protected $organizationName;
+    protected $organizationId;
+    protected $organizationAvatarUrl;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->webhookService = Mockery::mock(GithubWebhookService::class);
+        $this->webhookController = new GithubWebhookController($this->webhookService);
+
+        $this->user = User::factory()->create(["github_id" => 123]);
+        $this->organization = Organization::factory()->create(["github_id" => 456]);
+        $this->user->organizations()->attach($this->organization->id);
+
+        $this->organizationName = "test";
+        $this->organizationId = 123;
+        $this->organizationAvatarUrl = "http://example.com/avatar.png";
+    }
+
+    protected function tearDown(): void
+    {
+        if ($container = Mockery::getContainer()) {
+            $this->addToAssertionCount($container->mockery_getExpectationCount());
+        }
+        Mockery::close();
+        parent::tearDown();
+    }
+
+    public function testInvokeWithCreatedAction(): void
+    {
+        $request = Request::create("/webhook", "POST", [
+            "action" => "created",
+            "installation" => [
+                "account" => [
+                    "type" => config("services.organization.type"),
+                    "login" => "test",
+                    "id" => 123,
+                    "avatar_url" => "http://example.com/avatar.png",
+                ],
+            ],
+        ]);
+
+        $this->webhookService->shouldReceive("createOrganization")
+            ->once()
+            ->with("test", 123, "http://example.com/avatar.png");
+
+        $this->webhookController->__invoke($request);
+    }
+
+    public function testInvokeWithIncorrectType(): void
+    {
+        $request = Request::create("/webhook", "POST", [
+            "action" => "created",
+            "installation" => [
+                "account" => [
+                    "type" => "wrong_type",
+                    "login" => "test",
+                    "id" => 123,
+                    "avatar_url" => "http://example.com/avatar.png",
+                ],
+            ],
+        ]);
+
+        $this->webhookService->shouldNotReceive("createOrganization");
+
+        $this->webhookController->__invoke($request);
+    }
+
+    public function testInvokeWithMemberRemovedAction(): void
+    {
+        $request = Request::create("/webhook", "POST", [
+            "action" => "member_removed",
+            "organization" => [
+                "id" => 123,
+            ],
+            "membership" => [
+                "user" => [
+                    "id" => 456,
+                ],
+            ],
+        ]);
+
+        $this->webhookService->shouldReceive("removeMember")
+            ->once()
+            ->with(123, 456);
+
+        $this->webhookController->__invoke($request);
+    }
```

Comment:
I am thinking about these tests. I do not think so there are needed if we have the same tests below but I do not know.

Reply:
If u want test things like this:
```
$this->webhookService->shouldReceive("createOrganization")
            ->once()
            ->with("test", 123, "http://example.com/avatar.png");
```
I think, u can do that in tests below.

Reply:
deleted these tests